[pt] Initialize translations for /pt/docs/languages/go (#5346)
open-telemetry/opentelemetry.io: content/pt/docs/languages/go/getting-started.md

---
title: Primeiros Passos
weight: 10
default_lang_commit: 7fd0d2a6b87d6bbf2d5a35340c7afbd2bb33ca1c
# prettier-ignore
cSpell:ignore: chan fatalln funcs intn itoa khtml otelhttp rolldice stdouttrace strconv
---

<!-- markdownlint-disable blanks-around-fences -->
<?code-excerpt path-base="examples/go/dice"?>

Esta página mostrará como começar a utilizar o OpenTelemetry em Go.

Você aprenderá como instrumentar manualmente uma aplicação simples, de modo em

> que os [rastros][traces], [métricas][metrics] e [logs][] sejam emitidos no

Author: Feito!

  "SeverityText": "",
  "Body": {
    "Type": "String",
    "Value": "Alice está lançando os dados"
  },
  "Attributes": [
    {
      "Key": "result",
      "Value": {
        "Type": "Int64",
        "Value": 6
      }
    }
  ],
  "TraceID": "829fb7ceb787403c96eac3caf285c965",
  "SpanID": "8b6b408b6c1a35e5",
  "TraceFlags": "01",
  "Resource": [
    {
      "Key": "service.name",
      "Value": {
        "Type": "STRING",
        "Value": "dice"
      }
    },
    {
      "Key": "service.version",
      "Value": {
        "Type": "STRING",
        "Value": "0.1.0"
      }
    },
    {
      "Key": "telemetry.sdk.language",
      "Value": {
        "Type": "STRING",
        "Value": "go"
      }
    },
    {
      "Key": "telemetry.sdk.name",
      "Value": {
        "Type": "STRING",
        "Value": "opentelemetry"
      }
    },
    {
      "Key": "telemetry.sdk.version",
      "Value": {
        "Type": "STRING",
        "Value": "1.19.0-rc.1"
      }
    }
  ],
  "Scope": {
    "Name": "rolldice",
    "Version": "",
    "SchemaURL": ""
  },
  "DroppedAttributes": 0
}
```

</details>

Atualize a página <http://localhost:8080/rolldice/Alice> algumas vezes, e então

> você pode esperar um pouco ou encerrar a execução do aplicativo e verá as

Reviewer: ```suggestion
você pode esperar um pouco ou encerrar a execução da aplicação e verá as
```

métricas como na saída do console. Você verá a métrica `dice.rolls` emitida no

> console, com contagens separadas para cada valor de rolagem, bem como as

Reviewer: ```suggestion
console, com contagens distintas para cada valor obtido, bem como as
```

Não que esteja errado, mas achei um pouco esquisito, então fica a sugestão :)

                    }
                  },
                  {
                    "Key": "net.host.port",
                    "Value": {
                      "Type": "INT64",
                      "Value": 8080
                    }
                  }
                ],
                "StartTime": "2023-09-25T12:42:04.279219438+02:00",
                "Time": "2023-09-25T12:42:15.482697158+02:00",
                "Count": 27,
                "Bounds": [
                  0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000,
                  7500, 10000
                ],
                "BucketCounts": [
                  0, 27, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
                ],
                "Min": {},
                "Max": {},
                "Sum": 2.1752759999999993
              }
            ],
            "Temporality": "CumulativeTemporality"
          }
        }
      ]
    }
  ]
}
```

</details>

## Próximos passos {#next-steps}

Para mais informações sobre como instrumentar seu código, consulte a
documentação de [instrumentação manual](/docs/languages/go/instrumentation/).

Você também vai querer configurar um Exporter apropriado para

> [exportar os seus dados telemétricos](/docs/languages/go/exporters/) para um ou

Reviewer: ```suggestion
[exportar os seus dados de telemetria](/docs/languages/go/exporters/) para um ou
```

mais backends de telemetria.

Caso queira explorar um exemplo mais complexo, dê uma olhada na
[demonstração do OpenTelemetry](/docs/demo/), que inclui o

> [Serviço de Checkout](/docs/demo/services/checkout/) baseado em Go, o

Reviewer: ```suggestion
[Serviço de Checkout](/docs/demo/services/checkout/), o
```

[Serviço de Catálogo de Produtos](/docs/demo/services/product-catalog/), e o

> [Serviço de Contabilidade](/docs/demo/services/accounting/)

Reviewer: ```suggestion
[Serviço de Contabilidade](/docs/demo/services/accounting/), baseados em Go
```
